### PR TITLE
STF-1530 Bonus and Retailer Loyalty Cards API

### PR DESCRIPTION
**Pull request by volhit**
[Jira Task](https://instamart.atlassian.net/browse/STF-1530)

### CODE REVIEW

File: source/includes/_bonus_programs.md.erb [hunk 1]
```diff
@@ -0,0 +1,357 @@
+# Бонусные программы
+
+## API возможных типов бонусных карт
```
Comment (oleglitvin) [inline source/includes/_bonus_programs.md.erb]: Список бонусных программ

File: source/includes/_bonus_programs.md.erb [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+Получить список доступных бонусных программ можно, выполнив запрос:
+
+`GET <%= config[:host] %>/bonus_programs`
+
+Вернется массив доступных бонусных программ следующей структуры:
+
+Параметр|Описание
+---|---
+id|Идентификатор бонусной программы
```
Comment (oleglitvin) [inline source/includes/_bonus_programs.md.erb]: `Идентификатор ` --> `ID`
Check other documents to become familiar with naming convention.

File: source/includes/_bonus_programs.md.erb [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+  -H "Authorization: Token token=#{TOKEN}"
+```
+> Ответ будет содержать следующие данные:
+
+```json
+  {
+    "bonus_cards": [{
+      "id": 121,
+      "number": "1234567890",
+      "bonus_program_id": 1
```
Comment (oleglitvin) [inline source/includes/_bonus_programs.md.erb]: We also need card name and icon.

File: source/includes/_bonus_programs.md.erb [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+points|Количество начисляемых баллов по карте
+bonus_card|Описание бонусной карты
+
+Структура объекта bonus_card:
+
+Параметр|Описание
+---|---
+id|Идентификатор бонусной карты
+number|Номер бонусной карты
+bonus_program_id|Идентификатор бонусной программы
```
Comment (oleglitvin) [inline source/includes/_bonus_programs.md.erb]: Get rid of duplicated response descriptions.

Comment (volhit) [reply source/includes/_bonus_programs.md.erb]: fixed

File: source/includes/_bonus_programs.md.erb (hunk 3 above)
Comment (volhit) [reply source/includes/_bonus_programs.md.erb]: fixed

File: source/includes/_bonus_programs.md.erb (hunk 2 above)
Comment (volhit) [reply source/includes/_bonus_programs.md.erb]: done

File: source/includes/_bonus_programs.md.erb (hunk 1 above)
Comment (volhit) [reply source/includes/_bonus_programs.md.erb]: done

**Pull request merged**